src/content/styleguide.mdx

```diff
@@ -254,6 +254,33 @@ const MyComponent: AppFunctionComponent<Props> = ({ property, optionalProperty =
 }
 ```
 
+## Avoid destructuring props if it isn't needed
+
+Try to keep it as simple as possible passing props. Avoid destructuring props if you can pass whole props or part of props by use the rest operator.
```

Comment:
```suggestion
Try to keep passing props as simple as possible. Avoid destructurization if you can pass the whole object or part of it by using the rest operator.
```

Just an editorial. 🤷

Reply:
Thanks! Unfortunately, I have a tendency to complicate descriptions.